asynction/mock_server.py

```diff
@@ -210,7 +213,9 @@ def from_spec(
         )
 
     def _register_handlers(
-        self, default_error_handler: Optional[ErrorHandler] = None
+        self,
+        default_error_handler: Optional[ErrorHandler] = None,
+        server_security: Optional[Sequence[SecurityRequirement]] = None,
```

Comment:
```suggestion
        server_security: Sequence[SecurityRequirement],
```
Based on the comment in `types.py`